prov/efa/src/efa_cq.c

```diff
@@ -281,35 +306,45 @@ static inline int efa_cq_set_ibv_cq_ex(struct efa_cq *cq, struct fi_cq_attr *att
 
 	/* To enable additional WC fields, create CQ with efadv_create_cq verb instead. */
 	cq->ibv_cq_ex = efadv_create_cq(cq->domain->device->ibv_ctx,
-									&init_attr_ex,
-									&efadv_cq_init_attr,
-									sizeof(efadv_cq_init_attr));
+						   &init_attr_ex,
+						   &efadv_cq_init_attr,
+						   sizeof(efadv_cq_init_attr));
+
+	if (cq->ibv_cq_ex) {
+		/* Support operations on EFA DV CQ */
+		cq->flags |= EFA_CQ_SUPPORT_EFADV_CQ;
+		return 0;
+	}
 
-	return cq->ibv_cq_ex ? 0 : -FI_ENOCQ;
+	/* This could be due to old EFA kernel module versions */
+	EFA_INFO(FI_LOG_CQ, "Unable to create EFA DV CQ: %s\n", strerror(errno));
```

Comment:
I think `EFA_WARN` is better. It should not be something happen commonly for user.

Reply:
I kinda agree. I used `EFA_WARN` at the beginning but that made the unit test log noisy. Do you have a strong preference?

Reply:
I prefer EFA_WARN. I am ok that we see some warning message when running unit test.

Reply:
Addressed